.github/ISSUE_TEMPLATE/02-documentation.md

```diff
@@ -0,0 +1,13 @@
+---
+name:  "Handsontable Documentation"
```

Comment:
```suggestion
name: "Handsontable Documentation"
```
Unnecessary white space :)

Reply:
Done in a2ffafd2cae43a9cd57bdc3cb0c2e7d22c755ded. Thanks for noticing this.